apps/condo/domains/property/components/panels/Builder/MapConstructor.ts

```
@@ -357,97 +347,75 @@ class MapView extends Map {
+        return []
     }
 
-    get sections (): BuildingSection[] {
-        return this.map.sections
+    get defaultUnitType (): BuildingUnitSubType {
+        if (this.viewMode === MapViewMode.section) {
+            return BuildingUnitSubType.Flat
+        } else if (this.viewMode === MapViewMode.parking) {
+            return BuildingUnitSubType.Parking
+        }
     }
```

Biome (1.9.4)

[error] This getter should return a value.



(lint/suspicious/useGetterReturn)